src/main/java/com/limechain/rpc/methods/author/dto/DecodedKey.java

```diff
@@ -0,0 +1,14 @@
+package com.limechain.rpc.methods.author.dto;
+
+import com.limechain.storage.crypto.KeyType;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
```

Comment:
```suggestion
@Value
```
Seems like this only uses the all arg ctor and could have final for all fields as they never change.